From: Jason Wessel <jason.wessel@windriver.com>
Date: Sat, 2 Jul 2016 09:53:10 -0500
Subject: [PATCH] lxc-console: Silence output when not using an escape char

The lxc-console should not emit a message when the escape is set to -1.
Other places in the code already check for the escape -1
condition and properly disable the escape sequence.

Signed-off-by: Jason Wessel <jason.wessel@windriver.com>

---
 src/lxc/console.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/src/lxc/console.c
+++ b/src/lxc/console.c
@@ -667,7 +667,8 @@ int lxc_console(struct lxc_container *c,
 		goto err1;
 	}
 
-	fprintf(stderr, "\n"
+	if (escape != -1)
+		fprintf(stderr, "\n"
 			"Connected to tty %1$d\n"
 			"Type <Ctrl+%2$c q> to exit the console, "
 			"<Ctrl+%2$c Ctrl+%2$c> to enter Ctrl+%2$c itself\n",
